src/main/resources/application.properties

```diff
@@ -15,3 +15,6 @@ spring.jpa.show-sql=true
 spring.jpa.open-in-view=false
 spring.jpa.properties.hibernate.format_sql=true
 spring.jpa.properties.hibernate.highlight_sql=true
+
+jwt.expiration=300000
+jwt.secret=my-32-character-ultra-secure-and-ultra-long-secret
```

Comment:
It's better to get sensitive properties from system properties and not pushing it to git.